tests/setup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Project:  MapCache
+# Purpose:  MapCache tests
+# Author:   Even Rouault
```

Comment:
Jerome Boué

Reply:
Here, the original file is tests/travis_setup.sh.